src/pages/lastPage.js

```diff
@@ -0,0 +1,66 @@
+import { getQuizData } from '../utils/sessionStorage.js';
+
+export function lastPage() {
+  document.body.innerHTML = '';
+  let correctAnswers = 0;
+  let wrongAnswers = 0;
+  let skippedAnswers = 0;
+  for (let finalQuestions of getQuizData().questions) {
```

Comment:
Can you rename this as finalQuestion as it should be single

Reply:
done!